guides/advanced_guides/configuration.md

```diff
@@ -287,39 +295,43 @@ This command will stop the process if:
 
 If this command is not called, no snapshot will be imported.
 
-### Ignore missing snapshot
+### Ignore Missing Snapshot
 
 **Environment variable**: `MEILI_IGNORE_MISSING_SNAPSHOT`
 **CLI option**: `--ignore-missing-snapshot`
 
 The engine ignores missing snapshots and does not throw an error in this case.
 
-### Ignore snapshot if db exists
+Requires `--import-snapshot` to be defined.
+
+### Ignore Snapshot If DB Exists
 
 **Environment variable**: `MEILI_IGNORE_SNAPSHOT_IF_DB_EXISTS`
 **CLI option**: `--ignore-snapshot-if-db-exists`
 
 The engine skips snapshot importation if a database already exists. No error is thrown in this case.
```

Comment:
```suggestion
The engine skips snapshot importation if a database already exists, in the case of a `--snapshot-import`. No error is thrown in this case.
```

Reply:
```suggestion
The engine skips snapshot importation if a database already exists, in the case of a `--import-snapshot`. No error is thrown in this case.
```

?

Reply:
I want to make it clear that this variable has to do with the import snapshot process. You don't like it ?